.github/workflows/pre_release.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish to Test PyPI
+on:
+  push:
+    tags:
+      - '*.*.*'
+
+jobs:
+  test_pypi_release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - name: Install Poetry
+        run: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/get-poetry.py | python
```

Comment:
```suggestion
        run: curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -- -y
```

Reply:
This does not work for me:

```
$ curl -sSL https://raw.githubusercontent.com/python-poetry/poetry/master/install-poetry.py | python -- -y
/System/Library/Frameworks/Python.framework/Versions/2.7/Resources/Python.app/Contents/MacOS/Python: can't open file '-y': [Errno 2] No such file or directory
curl: (23) Failed writing body (0 != 1362)
```

Reply:
this fails on GH Actions as well: https://github.com/bbelderbos/biteme/runs/3813454151?check_suite_focus=true